.github/workflows/ci-webdriver.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       - name: 'Run Test'
         run: |
           pip install -r requirements.txt
-          python main.py --url="https://dev.dual.finance" --seed="$SEED" --password="$PASSWORD" --token="SOL"
           python main.py --url="https://beta.dual.finance?skipIpCheck=true" --seed="$SEED" --password="$PASSWORD" --token="SOL"
+          python main.py --url="https://dev.dual.finance" --seed="$SEED" --password="$PASSWORD" --token="SOL"
```

Comment:
this change isnt needed?

Reply:
probably not.
if there is something wrong with both, would rather not be spending real gas.